src/lib/storage/zero_suppression/zs_type.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <cstdint>
+
+namespace opossum {
+
+/**
+ * @brief Implemented zero suppression schemes
+ *
+ * Zero suppression is also known as null suppression
+ */
+enum class ZsType : uint8_t { Invalid, FixedSizeByteAligned, SimdBp128 };
```

Comment:
This would be a candidate to join with `encode_by_zs_type.hpp` as a central interface for the ZS, so that we limit the number of places that know about all compression schemes.

Reply:
done